assets/css/homepage.css

```diff
@@ -0,0 +1,530 @@
+
+.speakers-header h2 {
+  color: #272a31;
+}
+
+.speakers-header h2::after {
+  margin-left: 75px;
+}
+
+.speakers {
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-gap: 30px;
+  width: 100%;
+}
+
+.speaker {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+}
+
+.speaker-img {
+  background: url('../img/picsbg.JPG') no-repeat top left;
+  background-size: 50% 50%;
+  padding-left: 12px;
+  padding-top: 12px;
+  margin-right: 15px;
+}
+
+.speaker-img img {
+  width: 120px;
+  height: auto;
+}
+
+.speaker-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+}
+
+.speaker-name h5 {
+  color: #272a31;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 25px;
+  letter-spacing: 0;
+  margin-top: 12px;
+}
+
+.speaker-title h6 {
+  color: #ec5242;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 400;
+  font-size: 21px;
+  line-height: 22px;
+  letter-spacing: 0;
+}
+
+.speaker-name,
+.speaker-title {
+  margin-bottom: 15px;
+}
+
+.speaker-message p {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0;
+}
+
+@media screen and (max-width: 768px) {
+  #speakers {
+    padding: 30px 20px;
+  }
+
+  .speakers-header {
+    margin-bottom: 50px;
+  }
+
+  .speakers {
+    grid-template-columns: 100%;
+    grid-gap: 0;
+    grid-row-gap: 20px;
+  }
+
+  .speaker {
+    width: 100%;
+  }
+
+  .speaker-img {
+    padding-left: 14px;
+    padding-top: 14px;
+    margin-right: 12px;
+  }
+
+  .speaker-name h5 {
+    font-size: 21px;
+    line-height: 25px;
+  }
+
+  .speaker-title h6 {
+    font-size: 17px;
+    line-height: 20px;
+  }
+
+  .speaker-name,
+  .speaker-title {
+    margin-bottom: 5px;
+  }
+
+  .speaker-message p {
+    font-size: 14px;
+    line-height: 15px;
+  }
+}
+
+#partners-home {
+  /* background: #272a31; */
+  background: #000;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 40px 140px;
+  margin-top: 50px;
+}
+
+.partners-header h2::after {
+  margin-left: 22px;
+}
+
+.partners {
+  margin-top: 50px;
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.partners img {
+  height: 35px;
+  width: auto;
+  margin-right: 35px;
+}
+
+footer {
+  background: #d3d3d3;
+  padding: 50px 140px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer-logo {
+  margin-right: 40px;
+}
+
+.footer-logo img {
+  height: 40px;
+}
+
+.footer-info h5 {
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 500;
+  font-size: 18px;
+  margin-bottom: 12px;
+}
+
+.footer-info p {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly, please redesign your app using a mobile-first approach, as learned in the whole module that we finish now.